NEWS.md

```diff
@@ -1,76 +1,113 @@
- * PytestFramework in Travis (#2415)
- * syslog-ng-mod-java debian pkg should depend on headless jre (#2388)
- * Add contextual data error reporting improvements & csv-scanner refactor (#2373)
- * Afsocket remove unused functions/bitfields (#2363)
- * Afsocket minor cleanup/refactor (#2355)
- * Riemann worker (#2313)
- * Afsql threaded dest driver (#2097)
- * dbld: do not mount .gitconfig if missing (#2419)
- * dbld: Add missing docbook-xsl packages (#2398)
- * dbld: update criterion to 2.3.3 (#2396)
- * dbld: Remove "proposed" Ubuntu repository from enable_dbgsyms() (#2382)
- * dbld: Add new target "list-builder-images" (#2381)
- * dbld: Support Ubuntu Bionic and update existing images (#2318)
- * dbld: release target should use the default image (#2464)
+ * Add persist-tool to file list (#2561)
+ * Autotools, cmake: add detection for pylint, nosetests (#2537)
+ * Autotools: relaxing python dependency requirement  (#2472)
+ * Cfg-parser: printing error positions in case of parse failure (#2455)
+ * Cmake: add missing detections (#2510)
+ * Collect-cov.sh: make coverage should work with lcov in bionic (#2515)
+ * Configure: fix "--with-docbook*" option usage (#2465)
+ * Custom LGTM.com query for detecting uses of gmtime that are not covered by a lock (#2413)
+ * Fix static uClibc-ng support (#2501)
+ * Format non-literal fixes (#2567)
+ * Grammar: remove the pragma less include (#2550)
+ * Http auth header *(#2551)
+ * Http-worker: refactor & fix curl_easy_getinfo error message (#2527)
+ * Improve nosetests, pylint detection in case of python3 (#2564)
+ * Lib/compat: replace strcasestr() implementation (#2482)
+ * Libtest: Adding stopwatch.h into libtest_HEADERS list (#2553)
+ * Logthrdestdrv: factor out _process_result_xxx functions (#2528)
+ * Packaging: fix the description of mod-examples (#2522)
+ * Python-debugger: fix macro completion (#2439)
+ * Python, java destination add already introduced retry (#2559)
+ * Redis: consume ref at set_command (#2530)
+ * Refactor timestamp parsing code to be less coupled with message parsing (#2427)
+ * Remove elastic v1 support (#2554)
+ * Simplify libtest queue utils (#2556)
+ * Syslog-ng.8.xml: remove unneeded default-modules section (#2475)
+ * Threaded-dest: make persist_name local (#2516)
+ * Timeutils.c split (#2539)
+ * Travis: refactor - use brew and apt addon instead of shell execution (#2467)
+ * Travis: use the latest Bison version (macOS) (#2529)
+ * Various fixes for issues reported by LGTM (#2524)
 
 ## Notes to the developers
 
- * PytestFramework: Add initial test framework (#1940)
+ * Example-msg-generator: num option (#2565)
+ * Own grammar support in generator plugin (#2552)
+ * ProtoClient: provide process_in function to logwriter (#2468)
+ * Remove unused submodules (#2525)
+ * Simpler names for WORKER_INSERT_RESULT_T in language bindings (#2506)
+ * Split xml-parser into xml-parser and xml-scanner (#2459)
+
+## Pytest framework
```

Comment:
Is Pytest/Light something that we separately want to keep track of?

I'm asking this because of the separate section. This way, for me, it sound like a separate git project with a separate release process (that we've agreed we don't want do).